File: TUDiReader/Item.m
//
//  Item.m
//  TUDiReader
//
//  Created by Martin Weißbach on 11/18/13.
//  Copyright (c) 2013 Technische Universität Dresden. All rights reserved.
//

#import "Item.h"

@implementation Item

- (NSString *)htmlSummary
{
    NSMutableString *htmlString = [NSMutableString new];
    [htmlString appendString:@"<!DOCTYPE html><html><head><meta name=\"viewport\" content=\"initial-scale=1.0\"/>"];
    [htmlString appendString:@"<style type=\"text/css\">img {max-width:100%%;width:auto;height:auto;}"];
    [htmlString appendString:@"body {margin:0px 10px 10px;line-height:1.5; font-family:'Helvetica Neue';}</style></head>"];
    [htmlString appendFormat:@"<body>%@</body></html>", self.summary];
    
    return [NSString stringWithString:htmlString];
}

@end

File: TUDiReaderToday/Item.swift
//
//  Item.swift
//  TUDiReader
//
//  Created by Martin Weissbach on 1/20/15.
//  Copyright (c) 2015 Martin Weissbach. All rights reserved.
//

import Foundation
import CoreData

class Item : NSManagedObject {
    
    @NSManaged var title: String
    @NSManaged var author: String
    @NSManaged var dateString: String
    @NSManaged var summary: String
    @NSManaged var guid: String
    @NSManaged var read: Bool
    
    @NSManaged var feed: Feed
    
}